### DISCUSSION TITLE
eclipse symfony plugin

### DISCUSSION THREAD

**Discussion by kibjob50** [symfony/symfony#42671] [Q&A]
Ciao, tried to install from market place but ..

```
The following solutions are not available: Symfony Plugin 1.6.0
(id=com.dubture.symfony.feature.feature.group, org.dadacoalition.yedit.feature.feature.group,
site=http://p2.pdt-extensions.org/)
```

Can anybody tell me how and where get the plugin
Thanks in advance
Claudio

**Comment by lyrixx** [comment]
did you try https://marketplace.eclipse.org/content/symfony-plugin  ?

**Comment by javiereguiluz** [comment]
According to this comment from Eclipse Marketplace: https://marketplace.eclipse.org/comment/7635#comment-7635, this plugin is no longer installable because the related repository apparently is no longer available.

If you can afford to switch IDEs, PhpStorm and its Symfony plugin will provide you an excellent alternative solution.
